apps/chat/tests/test_tools.py

```diff
@@ -150,3 +151,39 @@ def test_create_schedule_message_experiment_does_not_exist():
         ).count()
 
         assert scheduled_message_count == 0
+
+
+@pytest.mark.django_db()
+def test_delete_schedule_tool():
+    session = ExperimentSessionFactory()
+    with freeze_time("2024-01-01"):
+        params = {"time_period": "days", "frequency": 1, "repetitions": 2, "prompt_text": "", "name": "Testy"}
+        system_scheduled_message = ScheduledMessage.objects.create(
+            participant=session.participant,
+            team=session.team,
+            action=EventActionFactory(params=params),
+            experiment=session.experiment,
+        )
+        user_scheduled_message = ScheduledMessage.objects.create(
+            participant=session.participant,
+            team=session.team,
+            experiment=session.experiment,
+            custom_schedule_params=params,
+        )
+
+        tool = DeleteReminderTool(experiment_session=session)
+
+        # User should not be able to delete this one
```

Comment:
nit: separate each of these cases into separate tests

Reply:
https://github.com/dimagi/open-chat-studio/pull/597/commits/7a809a0ac545069429b270c70c09018d57209592